GsaGH/Helpers/GsaAPI/Mappings.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace GsaGH.Helpers.GsaApi {
   internal static class Mappings {
-    internal static readonly Dictionary<string, AlignmentType> alignmentTypeMapping
+    internal static readonly Dictionary<string, AlignmentType> AlignmentTypeMapping
```

Comment:
this should be lower case
![image](https://github.com/arup-group/GSA-Grasshopper/assets/83698580/550343ae-29b6-42cd-b595-ed6c3514f736)

Reply:
changed